src/store/selectors/positions.ts

```diff
@@ -54,22 +54,27 @@ export const positionsWithPoolsData = createSelector(
       }
     })
 
-    return list.map((position, index) => {
+    const result = []
+    for (let index = 0; index < list.length; index++) {
+      const position = list[index]
       const tokenX = tokens.find(token => token.assetAddress === position.poolKey.tokenX)
       const tokenY = tokens.find(token => token.assetAddress === position.poolKey.tokenY)
 
       if (!tokenX || !tokenY) {
-        throw new Error(`Token not found for position: ${position}`)
+        console.log('Token not found for position: ', stringifyWithBigInt(position))
```

Comment:
i'm not sure if we should create function just to print something in console one time, you can print address of the missing token